user-manual/transfer/scripts/rights-mets.xml

```diff
@@ -16,8 +16,8 @@
           <premis:licenseTerms>Terms of license.</premis:licenseTerms>
           <premis:licenseNote>migrate</premis:licenseNote>
           <premis:licenseApplicableDates>
-            <premis:startDate>2000-09-09</premis:startDate>
-            <premis:endDate>2010-09-08</premis:endDate>
+            <premis:startDate>09-09-2000</premis:startDate>
```

Comment:
I notice the date formatting switched around (LibreOffice?) I might have missed this in the sample-data, I guess there's not a big difference when it comes to trying to consume this data from the METS eventually? And clearly Archivematica doesn't have a problem with it.